src/backend/core/cart/views.py

```diff
@@ -95,3 +121,212 @@ def delete(self, request, token, sku):
             return Response(status=HTTP_204_NO_CONTENT)
         except (Cart.DoesNotExist, CartItem.DoesNotExist):
             return Response(status=HTTP_404_NOT_FOUND)
+
+
+class PaymentMethodListDashboardView(GenericAPIView):
```

Comment:
We can probably use [`ListCreateApiView`](https://www.django-rest-framework.org/api-guide/generic-views/#examples) here, can't we?